notebooks/visualize/visualize.ipynb

```diff
@@ -0,0 +1,1034 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    graph.title.align = 'center'
</pre><p>You can change the xaxis label orientation, which might help with long items like this - maybe a shallow angle would be more readable than vertical</p><pre class="ql-syntax" spellcheck="false">p.xaxis.major_label_orientation = 45

# or alternatively:
p.xaxis.major_label_orientation = "vertical"
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/mstic-dataexpl/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>